arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -52,10 +52,10 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}": {
       "get": {
         "tags": [
-          "Registries"
+          "RegistriesGA"

```

Comment:
This needs to be the resource type "Registries". The same for other tags and operationId.